rest_api/tests/api_test/get/test_rest_api_get_batch.py

```diff
@@ -0,0 +1,285 @@
+#                       
+#         block_list = setup_batch[0]
+#         block_ids =  setup_batch[1]
+#         batch_ids =  setup_batch[2]
+#         expected_head_id = setup_batch[3]
+#         expected_id = batch_ids[0]
+#         start = -1
+#                      
+#         try:  
+#             response = get_batches(start=start)
+#         except urllib.error.HTTPError as error:
+#             data = json.loads(error.fp.read().decode('utf-8'))
+#             LOGGER.info(data['error']['title'])
+#             LOGGER.info(data['error']['message'])
+#             assert data['error']['code'] == 54
+#       
+#     def test_api_get_batch_list_invalid_limit(self):   
+#         """Tests that GET /batches is unreachable with bad limit parameter 
+#         """
+#         LOGGER.info("Starting test for batch with bad limit parameter")
+#                       
+#         block_list = setup_batch[0]
+#         block_ids =  setup_batch[1]
+#         batch_ids =  setup_batch[2]
+#         expected_head_id = setup_batch[3]
+#         expected_id = batch_ids[0]
+#         limit = 0
+#                  
+#         try:  
+#             response = get_batches(limit=limit)
+#         except urllib.error.HTTPError as error:
+#             data = json.loads(error.fp.read().decode('utf-8'))
+#             LOGGER.info(data['error']['title'])
+#             LOGGER.info(data['error']['message'])
+#             assert data['error']['code'] == 53
+#                  
+#     def test_api_get_batch_list_reversed(self):   
+#         """Tests that GET /batches is unreachable with bad head parameter 
+#         """
+#         LOGGER.info("Starting test for batch with bad head parameter")
+#                       
+#         block_list = setup_batch[0]
+#         block_ids =  setup_batch[1]
+#         batch_ids =  setup_batch[2]
+#         expected_head_id = setup_batch[3]
+#         expected_id = batch_ids[0]
+#         reverse = True
+#                      
+#         try:
+#             response = get_batches(reverse=reverse)
+#         except urllib.error.HTTPError as error:
+#             assert response.code == 400
+#                     
+#         assert response['head'] == expected_head_id , "request is not correct"
+#         assert response['paging']['start'] == None ,  "request is not correct"
+#         assert response['paging']['limit'] == None ,  "request is not correct"
+#         assert bool(response['data']) == True
+#          
+# class BatchGetTest():
+#     def test_api_get_batch_id():
+#         """Tests that transactions are submitted and committed for
+#         each block that are created by submitting intkey batches
+#         """
+#         LOGGER.info('Starting test for batch post')
+#         LOGGER.info("Creating batches")
+#         batches = make_batches('abcd')
+#             
+#         LOGGER.info("Submitting batches to the handlers")
+#             
+#         for i, batch in enumerate(batches):
```

Comment:
where was this variable "batches" received from? i don't see it being received in class.